packages/components/radio/stories/radio.stories.tsx

```
@@ -205,6 +206,35 @@ const ControlledTemplate = (args: RadioGroupProps) => {
 };
 
+const ServerValidationTemplate = (args: RadioGroupProps) => {
+  const [serverErrors, setServerErrors] = React.useState({});
+  const onSubmit = (e) => {
+    e.preventDefault();
+    setServerErrors({
+      option: "You must choose an option.",
+    });
+  };
+
+  delete args.isInvalid;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)